asv_bench/benchmarks/groupby.py

```diff
@@ -493,6 +493,33 @@ def time_frame_agg(self, dtype, method):
         self.df.groupby("key").agg(method)
 
 
+class CumminMax:
+    param_names = ["dtype", "method"]
+    params = [
+        ["float64", "int64", "Float64", "Int64"],
+        ["cummin", "cummax"],
+    ]
+
+    def setup(self, dtype, method):
```

Comment:
is this costly?  worth using setup_cache?

Reply:
Looks to take ~0.25s. So might be worth caching, but appears `setup_cache` can't be parameterized, so would have to ugly up the benchmark a bit.

Reply:
yep have shrunk benchmark